src/test/java/com/exasol/adapter/dialects/exasol/ExasolColumnMetadataReaderTest.java

```diff
@@ -22,22 +22,6 @@ void beforeEach() {
                 BaseIdentifierConverter.createDefault());
     }
 
-    @Test
-    void testMapJdbcTypeIntervalDayToSeconds() {
-        final JdbcTypeDescription jdbcTypeDescription = new JdbcTypeDescription(
-                ExasolColumnMetadataReader.EXASOL_INTERVAL_DAY_TO_SECONDS, 0, 0, 0, "INTERVAL_DAY_TO_SECONDS");
-        assertThat(this.exasolColumnMetadataReader.mapJdbcType(jdbcTypeDescription),
-                equalTo(DataType.createIntervalDaySecond(2, 3)));
-    }
-
-    @Test
-    void testMapJdbcTypeIntervalYearToMonths() {
-        final JdbcTypeDescription jdbcTypeDescription = new JdbcTypeDescription(
-                ExasolColumnMetadataReader.EXASOL_INTERVAL_YEAR_TO_MONTHS, 0, 0, 0, "INTERVAL_YEAR_TO_MONTHS");
-        assertThat(this.exasolColumnMetadataReader.mapJdbcType(jdbcTypeDescription),
-                equalTo(DataType.createIntervalYearMonth(2)));
-    }
-
```

Comment:
I removed these tests since they do not test something relevant anymore. The relevant part can obly be tested by the integration tests.